src/modules/jobs/job.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable, InternalServerErrorException, Logger } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Job } from './job.entity';
+import { PaginationResult } from './pagination-result.interface';
+
+@Injectable()
+export class JobService {
+  private readonly logger = new Logger(JobService.name);
+
+  constructor(
+    @InjectRepository(Job)
+    private jobRepository: Repository<Job>
+  ) {}
+
+  async findAll(page: number = 1, pageSize: number = 10): Promise<PaginationResult<Job>> {
```

Comment:
make this more descriptive like saying getAllJobs

Reply:
Done sir 🤧